packages/cli/src/commands/init.ts

```diff
@@ -569,6 +581,7 @@ async function createTriggerPageRoute(
   const extension = isTypescriptProject ? ".ts" : ".js"
   const triggerFileName = `trigger${extension}`
   const examplesFileName = `examples${extension}`
+  const examplesIndexFileName = `index${extension}`
 
   const routeContent = `
 import { createPagesRoute } from "@trigger.dev/nextjs";
```

Comment:
You need to add the import for the Jobs file to this pages route as well:

`import "@/jobs";`

Reply:
Added jobs import to routeContent for createTriggerPageRoute function